pkg/maglev/maglev.go

```diff
@@ -74,6 +85,10 @@ func getPermutation(backends []string, m uint64, numCPU int) []uint64 {
 	// scheduling, etc) instead of a performance gain.
 
 	bCount := len(backends)
+
+	// As the permutations array is large and often used, we'll keep a single
+	// instance around and reuse it.
+	permutation := permutations.get()
 	if size := uint64(bCount) * m; size > uint64(len(permutation)) {
```

Comment:
This is going to reallocate for each increment of `bCount`. The previous solution never reallocated in practice and in theory it would only reallocate on the increase of `m`.